src/Fantomas/Program.fs

```diff
@@ -82,58 +88,64 @@ let rec allFiles isRec path =
 /// Fantomas assumes the input files are UTF-8
 /// As is stated in F# language spec: https://fsharp.org/specs/language-spec/4.1/FSharpSpec-4.1-latest.pdf#page=25
 let private hasByteOrderMark file =
-    if File.Exists(file) then
-        let preamble = Encoding.UTF8.GetPreamble()
+    async {
```

Comment:
Converted many functions in this file to `async` and used `Async` variants of all stream/file methods.

Reply:
Makes sense!